packages/client/src/components/SafeOverview.js

```diff
@@ -1,103 +1,67 @@
+        .filter((token) => token.balance > 0)
+    : [];
+  if (tokensWithPositiveBalance.length > 3) tokensWithPositiveBalance = tokensWithPositiveBalance.slice(0, 3);
+
   return (
     <>
       <div className="column p-0 mt-5 is-flex is-full">
         <h4 className="is-size-5">Safe Overview</h4>
       </div>
       <div className="column p-0 mt-4 mb-5 is-flex is-full">
-        <div className={balanceClasses.join(" ")}>
-          <div className="column is-full p-0 is-flex is-justify-content-space-between mb-2">
-            <div>Available</div>
-            <div>FLOW</div>
-          </div>
-          <div className="column is-full p-0 mb-2">
-            <h2 className="is-size-4">{flowBalance}</h2>
-          </div>
-          <BalanceBar progress={100} />
-          <div className="column is-full p-0 is-flex is-justify-content-space-between mt-2 is-size-6">
-            {flowBalance > 0 ? (
-              <div>100% FLOW</div>
-            ) : (
-              <>
-                <div>Your safe is empty</div>
-                <div className="is-underlined">Transfer assets</div>
-              </>
-            )}
-          </div>
-        </div>
         <div
-          className="rounded-sm border-light has-shadow p-5 is-flex"
-          style={{ minWidth: 375 }}
+          className={balanceClasses.join(" ")}
+          style={{ overflow: "hidden", minHeight: "200px", position: "relative" }}
         >
-          <div
-            className={[...moneyClasses, "border-light-right pr-5"].join(" ")}
-          >
-            <div>
-              <div>Money in</div>
-              <div className="has-text-grey is-size-6 mt-1">
-                Over the last 30-days
-              </div>
+          <LogoV style={{ position: "absolute", top: "0px", right: "-30px", zIndex: 0 }} />
+          <div style={{ zIndex: 1 }}>
+            <div className="column is-full p-0 mb-5 is-flex is-flex-direction-row is-justify-content-space-between">
+              <label>Tokens</label>
+              {!!tokensWithPositiveBalance.length && (
+                <Link to={(location) => `${location.pathname}/assets`}>View All</Link>
+              )}
             </div>
-            <div>
-              <div className="is-size-4">$0.00</div>
-            </div>
-          </div>
-          <div className={[...moneyClasses, "pl-5"].join(" ")}>
-            <div>
-              <div>Money out</div>
-              <div className="has-text-grey is-size-6 mt-1">
-                Over the last 30-days
+            {!tokensWithPositiveBalance.length ? (
+              <div className="p-3 m-1">
+                <div className="is-flex is-flex-direction-column is-justify-content-center is-align-items-center">
+                  <p className="ml-3 has-text-white ">You don't have any tokens</p>
+                  <button className="button column is-half is-medium has-background-white mt-5">Deposit Tokens</button>
+                </div>
```

Comment:
This button CTA should land user on Deposit flow, the work to implement deposit flow is tracked under a different ticket: https://linear.app/dappercollectives/issue/VSL-121. cc @KsenijaZivkovic

Reply:
Got it! Should we have it hooked up with the deposit token until function (in the testtollbox) or leave it as it for right now?

Reply:
let's leave it for now. once Deposit flow is ready we can then hook it up.